package.json

```diff
@@ -6,7 +6,7 @@
   "author": {
     "name": "Innogames GmbH"
   },
-  "version": "0.17.1",
+  "version": "0.17.2",
```

Comment:
this shouldn't be changed in the PR. After merging the PR we can create the new version including this change